### PR TITLE
Added generation of .env file

### PR DESCRIPTION
**Pull request by scriptcoded**
In response to #11 

Happy Hacktoberfest!

### CODE REVIEW

**Comment by Fabianopb** [comment]
Thanks @scriptcoded I'll check it later today! :)

File: create-mern-ts-app.js [hunk 1]
```diff
@@ -43,6 +44,9 @@ function createProjectTemplate(projectName) {
   console.log(chalk.cyan(destRoot + '\n'));
   fs.mkdirsSync(destRoot);
   fs.copySync(srcRoot, destRoot);
+  let envContents = fs.readFileSync(path.join(destRoot, 'backend', '.env.example'), 'utf8')
+    .replace(/auth-shared-secret/g, crypto.randomBytes(48).toString('hex'));
```
Comment (Fabianopb) [inline create-mern-ts-app.js]: I think some `24` bytes would already be quite enough here!

File: create-mern-ts-app.js [hunk 2]
```diff
@@ -43,6 +44,9 @@ function createProjectTemplate(projectName) {
   console.log(chalk.cyan(destRoot + '\n'));
   fs.mkdirsSync(destRoot);
   fs.copySync(srcRoot, destRoot);
+  let envContents = fs.readFileSync(path.join(destRoot, 'backend', '.env.example'), 'utf8')
```
Comment (Fabianopb) [inline create-mern-ts-app.js]: could use `const` instead of `let` as this variable is not reassigned.

Comment (Fabianopb) [inline create-mern-ts-app.js]: Would you mind doing two extra things here please, shouldn't take much work:

First would be to replace the comments on top of the generated `.env` file with something like
```
# This is an auto-generated file, change at your own will and risk.
```

Second, would you mind separating this into a new function, for example, `generateEnvFile()` to be called right after `createProjectTemplate` in the end of the script?

File: create-mern-ts-app.js (hunk 1 above)
Comment (scriptcoded) [reply create-mern-ts-app.js]: Done!

**Comment by scriptcoded** [comment]
Dumb me resolved all of your conversations. Sorry! I incorporated all of your changes, but the PR doesn't seem to update. Any ideas?

**Comment by Fabianopb** [comment]
That's funny, why the heck it's not updating if everything seems right in your fork? 🤔 
What if you try to re-create the PR?

**Comment by scriptcoded** [comment]
I'll try that

**Pull request closed without merging**